support/rollup/examples.js

```diff
@@ -0,0 +1,66 @@
+import path from 'path'
+import alias from 'rollup-plugin-alias'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import json from 'rollup-plugin-json'
+import replace from 'rollup-plugin-replace'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+import pkg from '../../package.json'
+
+const configurations = []
+
+const umdConfig = {
+  input: 'examples/index.js',
+  output: {
+    file: pkg.browser,
+    name: 'slate-examples',
+    format: 'umd',
+    exports: 'named',
+    sourcemap: process.env.ROLLUP_WATCH,
+  },
+  plugins: [
+    alias({
```

Comment:
Can you add a comment above this `alias` usage to explain what it does? It's not clear from looking at this why just these two modules are aliased?